uci-mode.el

```diff
@@ -202,6 +202,23 @@ form allows access to remote engines over SSH."
 
 ;;; Utility functions
 
+(defun uci-mode--get-engine-proc ()
+  "Return the engine process at `uci-mode-engine-buffer'."
+  (get-buffer-process uci-mode-engine-buffer))
+
+(defun uci-mode--process-alive-p ()
+  "Return true iff uci-mode engine process is alive."
+  (let ((proc (uci-mode--get-engine-proc)))
+    (and proc (process-live-p proc))))
+
+(defun uci-mode--get-engine-buffer ()
+  "Return the engine comint buffer using `uci-mode-engine-buffer-name'."
+  (get-buffer uci-mode-engine-buffer-name))
+
+(defun uci-mode--get-engine-buffer-window ()
```

Comment:
Seems incorrect that this is the same code as the above defun.

Reply:
Yep, that I stopped implementing that and forgot to delete. Removed.